static/js/popovers.js

```diff
@@ -226,6 +249,33 @@ exports.register_click_handlers = function () {
         show_message_info_popover(this, rows.id(row));
     });
 
+    var emoji_list = emoji.emojis_by_name;
```

Comment:
We should probably be fetching this dynamically when we actually are going to access it (i.e. in the call to render_emoji_template).  I think the current version will correctly handle the case where we extend emoji_list (maybe it will, but seems clearer regardless to just access this inside render_emoji_template()).